src/lib/ParsingHelpers.cs

```diff
@@ -36,6 +44,17 @@ internal static List<T> GetList<T>(JsonElement v, Func<JsonElement, T> load)
         return map;
     }
 
+    internal static Dictionary<string, string> GetMapOfString(JsonElement v)
+    {
+        var map = new Dictionary<string, string>();
+        foreach (var item in v.EnumerateObject())
+        {
+            var value = item.Value.GetString();
+            map.Add(item.Name, value ?? string.Empty);
```

Comment:
don't null coalesce strings when NRT is enabled, use ternary with string.isnullorempty (or whitespace)